packages/nuxt/src/core/templates.ts

```diff
@@ -334,6 +334,21 @@ export const publicPathTemplate: NuxtTemplate = {
   }
 }
 
+export const dollarFetchTemplate: NuxtTemplate = {
+  filename: 'fetch.mjs',
+  getContents () {
+    return [
+      "import { $fetch } from 'ofetch'",
+      "import { baseURL } from '#build/paths.mjs'",
+      'if (!globalThis.$fetch) {',
+      '  globalThis.$fetch = $fetch.create({',
+      '    baseURL: baseURL()',
+      '  })',
+      '}'
+    ].filter(Boolean).join('\n')
```

Comment:
Filtering plain strings might be redundant here
```suggestion
    ].join('\n')
```